### PR TITLE
Add nui

### PR DESCRIPTION
**Pull request by Sigmatecture**
Everything works except the 25% SpD drop on the foe when attracted. Not sure which handler I should use, or add it to nui's attract in conditions.ts.

### CODE REVIEW

File: data/mods/ssb/abilities.ts [hunk 1]
```diff
@@ -1323,6 +1323,31 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
+		desc: "On switch-in, this Pokemon attracts the opponents regardless of gender. Pokemon who are attracted have their Special Defense stat reduced by 25%.",
+		shortDesc: "Attracts opponent. Attracted Pokemon have SpD reduced by 25%.",
+		onStart(pokemon) {
+			const target = pokemon.side.foe.active[0];
+			target.addVolatile('attract');
+			this.add('-start', target, 'Attract', '[from] ability: Condition Override', '[of] ' + pokemon);
+
+			if (target.volatiles['attract']) {
+				this.boost({spd: -1}, target, pokemon, null, true)
+			}
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]: use onModifySpd in the attract status

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -2899,6 +2899,42 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		flags: {},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, "Celebrate", target);
+		},
+		onHit(target, source) {
+			const hax = ['slp', 'brn', 'par', 'tox'];
+			target.trySetStatus(this.sample(hax), source);
```
Comment (ghost) [inline data/mods/ssb/moves.ts]: make this.sample(hax) its own variable for the if chain below

Comment (Sigmatecture) [reply data/mods/ssb/moves.ts]: Fine with me, but what if the target is already statused? E.g. nui used Win Condition the turn prior and burned the foe. Then using WinCondition this turn will fail a new status but should default to Fire Blast as the attack

Comment (ghost) [reply data/mods/ssb/moves.ts]: shouldnt matter. It would just attempt to status then use the corresponding move

Comment (Sigmatecture) [reply data/mods/ssb/moves.ts]: ok that makes it less broken

File: data/mods/ssb/abilities.ts [hunk 3]
```diff
@@ -1323,6 +1323,20 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
...
 
+	// nui
+	conditionoverride: {
+		desc: "On switch-in, this Pokemon attracts the opponents regardless of gender. Pokemon who are attracted have their Special Defense stat reduced by 25%.",
+		shortDesc: "Attracts opponent. Attracted Pokemon have SpD reduced by 25%.",
+		onStart(pokemon) {
+			const target = pokemon.side.foe.active[0];
+			target.addVolatile('attract');
+			this.add('-start', target, 'Attract', '[from] ability: Condition Override', '[of] ' + pokemon);
+		},
```
Comment (ghost) [inline data/mods/ssb/abilities.ts]: nO

This should just give jiggly the ability to. Not auto do it on switch in pls

**Pull request merged**